pages/your_tasks/how_to_make_a_good_readme.md

```diff
@@ -0,0 +1,125 @@
+* Document the process to reproduce/replicate of experiments.
+* Citation information (it can be included in the file or in an additional file)
+* Relation with other research outputs such as publications, datasets, etc. These research outputs can be in the repository or in external repositories.
+
+
+### Considerations <!-- do not delete this heading and write your text below it -->
+* A good README favors the use of your software[^1].
+* A good README increments the transparency of your research.
+* A good README will help you to mantain the software during the software lifecycle.
+ 
+
+## Which sections should be addressed in a good README? <!-- example: how to name a software release? -->
+ 
+### Description <!-- do not delete this heading and write your text below it -->
+A README file is the first document that a researchers read when they are interested on using a software. For this reason, should contain all the relevant information to install and to execute it. It includes references to additional information such as API documentation, specifications, scientific publications related, tutorials, guidelines, etc.
+
+A basic software documentation emphasizes these aspects[^2] :
+* Understandability: a user can comprehend the structure and the purpose of the software.
+* Usability: a user can understant how to use the software.
+* Attribution: how a user must cite the software.
+
+These aspects should be addressed in different sections of the README. 
+
+### Considerations <!-- do not delete this heading and write your text below it -->
+* A good README should describe the purpose and the features of the software.
+* A good README should describe how to execute and to use the code. 
+* In a research context, a good README should indicate how the software must be cited.
+* In a research context, a good README should indicate the funding agency and the proper acknowledgements.
+* In the context of scientific experiments, a good README should facilitate its reproducibility.
+
+### Solutions <!-- do not delete this heading and write your text below it -->
+
+The following sections can be included in the README to facilitate the use of the software and to allow its citation.
+
+**Description**
+
+In this section, you will have to describe the purpose and the features of your software.
+
+**Requirements**
+
+In this section, you can include the requirements needed to install and to execute the software. These requirements can be software (e.g. operative system) or hardware (e.g. CPU, memory, etc.). There is no need to include low level requirements like language library dependencies. These requirements must be provided with the appropriate mechanism of each language. For example, in Python by using the requirements.txt file. 
+
+This is a non exhaustive list of requirements categories that you can consider:
+Operative System version
+Operative System libraries version
```

Comment:
```suggestion
Operating system libraries version
```

Reply:
Not sure what is meant by "Operating system libraries version" - is it the dependencies of the software?

Reply:
I understand this ad the packages installed in the operating system itself. For example, you must have cmake, you  must have a separate visualization software, etc.